tools/bug-report/pkg/bugreport/flags.go

```diff
@@ -92,12 +88,6 @@ func addFlags(cmd *cobra.Command, args *config2.BugReportConfig) {
 		"List of comma separated glob patters to match against log error strings. "+
 			"Any error matching these patters is ignored when calculating the log importance heuristic.")
 
-	// archive and upload control
-	cmd.PersistentFlags().StringVar(&args.GCSURL, "gcs-url", "",
-		"URL of the GCS bucket where the archive is uploaded.")
-	cmd.PersistentFlags().BoolVar(&args.UploadToGCS, "upload", false,
-		"Upload archive to GCS bucket. If gcs-url is unset, a new bucket is created.")
```

Comment:
Can you also remove `and optionally uploads the archive to a GCS bucket`?

https://github.com/istio/istio/blob/594b4d7bf9d11d242b842e356115d8fd7e4e7875/tools/bug-report/pkg/bugreport/bugreport.go#L65-L66

Reply:
Done.